### DISCUSSION TITLE
Fetch missing data

### DISCUSSION THREAD

**Discussion by henkdebruijn** [bassmaster187/TeslaLogger#1346] [Q&A] [answered]
I use Teslalogger Docker version on a Synology NAS. For a period of approximately a month no data is logged. Is there a way to resync with Tesla and fetch the missing data?

**Comment by rowich** [comment, marked as answer]
There is no way to fetch missing data - if a logger (regardless of the product) is not running, the data will be missing. You get it live or you get nothing.
